template/CONTRIBUTING.md.jinja

```diff
@@ -41,14 +41,14 @@ happy for any support that they can get.
 
 {{ project_name }} could always use more documentation, whether as
 part of the official documentation, in docstrings, or even on the web in blog
-posts, articles, and such. Just [open an issue]({{ dev_platform_url }}/{{ username }}/
-{{ project_slug }}/issues) to let us know what you will be working on
-so that we can provide you with guidance.
+posts, articles, and such. Just
+[open an issue]({{ dev_platform_url }}{{ username }}/{{ project_slug }}/issues)
```

Comment:
Several links are broken as they are defined across separate lines. This is a simple fix!